src/search.cpp

```diff
@@ -1011,7 +1011,9 @@ namespace {
 
       // Step 12. Pruning at shallow depth (~200 Elo)
       if (  !rootNode
-          && !(training && PvNode)
+#ifdef EVAL_LEARN
+          && !(!prune_at_shallow_depth_on_pv_node && PvNode)
```

Comment:
`&& (PvNode ? prune_at_shallow_depth_on_pv_node : true)`

Otherwise it breaks bench.

Reply:
Thank you for the suggestion.  The suggested code looks more intuitive.  I followed it.